sass/components/_view.scss

```diff
@@ -47,13 +47,15 @@
         .content {
             display: block;
             height: 145px;
-            padding: 0px 5px 16px 16px;
+            padding: 0 7px 0 16px;
+            position: relative;
 
             .ciq-list {
-                height: 115px;
-                overflow: auto;
-                display: block;
+                max-height: 110px !important;
 
+                .ciq-list-inner {
+                    min-height: 131px;
```

Comment:
this line is causing the scrollbar to show up in templates eventhough there's nothing there. Why did you add this?